docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
   redis:
     container_name: thatkitebot_redis
     volumes:
-      - ./data/redis:/data
+      - ./data/redis:/data:Z
```

Comment:
since this contains the redis data which is shared with another container, shouldn't it use `:z` instead? 
> The z option tells Podman that two containers share the volume content. As a result, Podman labels the content with a shared content label. Shared volume labels allow all containers to read/write content. The Z option tells Podman to label the content with a private unshared label. Only the current container can use a private volume.

(from the [podman github](https://github.com/containers/podman/blob/main/troubleshooting.md#2-cant-use-volume-mount-get-permission-denied))